mos-platform/atari8-stdcart/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+platform(atari8-stdcart COMPLETE PARENT atari8)
+
+if(NOT CMAKE_CROSSCOMPILING)
+  return()
+endif()
+
+install(FILES
+  cart-rom-fixed.ld
+  link.ld
+  stdcart.ld
+TYPE LIB)
+
+# TODO; refactor out atari8 targets
+add_platform_library(atari8-stdcart-crt0 syms.s)
+merge_libraries(atari8-stdcart-crt0
+  atari8-crt0
+  common-copy-data
```

Comment:
This causes two copies of the __copy_zp_data function to be present in this library: one inside common-copy-data (it does both), and one inside the merged atari8-crt0.

There's two ways to fix this:
1) Do the refactor to break out an atari8-common; then atari8 can just copy zp data, and atari8-stdcart can copy both zp data and regular data.
2) Make common-copy-data only copy .data, and not include ZP automatically. This is a breaking change, so you'd need to edit all other uses of common-copy-data and add common-copy-zp-data too. If you go this route, you should also update the porting wiki to reflect the new semantics of common-copy-data.